bundle/edu.gemini.spModel.io/src/main/scala/edu/gemini/spModel/io/impl/migration/to2015B/To2015B.scala

```diff
@@ -116,6 +194,36 @@ object To2015B {
     ps
 
   }
+
+  // Proper Motion conversion to degrees/yr
```

Comment:
I added comments in commit dd0375ef3c2a775f2235d9d635657a147e2d8e8a. It might be sensible to create numeric types that carry units, but that's more than I want to bite off right now.